components/subscribe.tsx

```diff
@@ -5,7 +5,7 @@ import { Row, Col, Form, Button } from "react-bootstrap";
 import { ErrorHandler } from "../utils/errorHandler";
 import { isBrowser } from "../utils/isBrowser";
 
-export default function Subscribe() {
+export default function Subscribe(props) {
```

Comment:
This works, but I the props can be defined like this to ensure that the campaign name is always required when the component is used: 

```suggestion
export default function Subscribe({campaign}:{campaign:string}) {
```